src/ReceivingList/utils.js

```diff
@@ -118,31 +118,16 @@ export const fetchOrderLineLocations = (ky) => (orderLines, fetchedLocationsMap)
 };
 
 export const fetchConsortiumOrderLineLocations = (ky, stripes) => (orderLines) => {
-  const tenantIds = [...new Set(
-    orderLines
-      .flatMap(({ locations }) => locations?.map(({ tenantId }) => tenantId))
-      .filter(Boolean),
-  )];
-
-  return chunkRequests(
-    tenantIds,
-    (tenantId) => fetchConsortiumInstanceLocations(
-      getConsortiumCentralTenantKy(ky, stripes),
-      {
-        searchParams: { tenantId },
-      },
-    ),
-  ).then((chunkedResponses) => {
-    const orderLinesLocationIdsSet = new Set(
-      orderLines.flatMap(({ locations }) => {
-        return map(locations, 'locationId');
-      }),
-    );
-
-    return chunkedResponses
-      .flatMap(({ locations }) => locations)
-      .filter(({ id }) => orderLinesLocationIdsSet.has(id));
-  });
+  return fetchConsortiumInstanceLocations(getConsortiumCentralTenantKy(ky, stripes))
```

Comment:
mod-search endpoints change context to central tenant automatically with any incoming tenantId. But it also fine to use cental tenant from UI request. Wrote just for reference